bin/tic-tac-toe.rb

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env ruby
+require '../lib/board.rb'
```

Comment:
Kindly check this. I tried running /bin/tic-tac-toe.rb and I'm getting the following error:

```
.....require': cannot load such file -- ../lib/board.rb
```
Probably it's better to use
`require_relative` instead of plain `require`. I didn't get the error anymore when I converted it into require_relative